packages/docusaurus-plugin-content-pages/src/index.js

```diff
@@ -47,10 +47,8 @@ module.exports = function(context, opts) {
 
       return pagesFiles.map(relativeSource => {
         const source = path.join(pagesDir, relativeSource);
-        const aliasedSource = path.join(
```

Comment:
This would not work if `pages` was for some reason placed outside the `siteDir`.